cosmos/dbt/project.py

```diff
@@ -21,6 +19,16 @@ def create_symlinks(project_path: Path, tmp_dir: Path, ignore_dbt_packages: bool
             os.symlink(project_path / child_name, tmp_dir / child_name)
 
 
+def copy_manifest_for_partial_parse(project_path: Path, tmp_dir: Path) -> None:
+    partial_parse_file = Path(project_path) / DBT_TARGET_DIR_NAME / DBT_PARTIAL_PARSE_FILE_NAME
+
+    if partial_parse_file.exists():
+        tmp_target_dir = tmp_dir / DBT_TARGET_DIR_NAME
+        tmp_target_dir.mkdir(exist_ok=True)
+
+        shutil.copy(str(partial_parse_file), str(tmp_target_dir / DBT_PARTIAL_PARSE_FILE_NAME))
```

Comment:
do we want to copy or symlink here?

Reply:
I'm like 98% sure you want to copy, not symlink. The reason is because the `partial_parse.msgpack` can be mutated by a dbt command invocation, and the symlink is only compatible with users' runtimes for read-only files. Because the `partial_parse.msgpack` can be overwritten, this means we cannot symlink to it.